cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/main/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizer.java

```diff
@@ -77,6 +109,7 @@ public void initialize(AuthorizationContext context) throws Exception {
 
   @Override
   public void grant(EntityId entityId, Principal principal, Set<Action> actions) throws RoleNotFoundException {
+    LOG.debug("Granting {} on {} to {}", actions, entityId, principal);
```

Comment:
It is generally helpful to find out the place where the call flow broke in case of an error so I will change it to trace but leave it here.